package.json

```diff
@@ -17,14 +17,25 @@
     "url": "https://github.com/dojo/cli-build-app.git"
   },
   "scripts": {
+    "artifact:build": "cd test-app && npm run build:dist:evergreen && shx mv output/dist output/dist-evergreen  && npm run build:dev:evergreen && shx mv output/dev output/dev-evergreen && npm run build:dist && npm run build:dev && npm run build:dist:evergreen",
+    "artifact:install": "cd test-app && shx rm -rf node_modules && npm i && npm run install-build-app",
+    "artifact:package": "cd dist && npm pack -q ./release && shx mv dojo-cli-build-app-* dojo-cli-build-app.tgz && cd ..",
+    "build:static:dev": "cpx \"{src,tests}/**/*.{d.ts,html,md,json,js,css}\" dist/dev",
+    "build:static:release": "cpx \"src/**/*.{d.ts,json}\" dist/release",
+    "build:cjs": "tsc",
+    "build": "concurrently \"npm:build:*\" && dojo-package",
+    "clean": "rimraf dist coverage test-app/output",
+    "lint": "tslint -p .",
+    "precommit": "lint-staged",
+    "prettier": "prettier --write 'src/**/*.ts' 'tests/**/*.ts'",
     "test:ci": "run-p --race serve test",
+    "test": "npm run build && npm run artifact:package && npm run artifact:install && npm run build && intern && npm run artifact:build && cypress run",
```

Comment:
We should add `prettier -l {src,tests}/**/*.ts` to fail the build if files don't comply